UKHomeOffice/sgmr-service
# SPCR-291 Update Cypress & fix add people tests

## Description

- update mailslurp details
- update syntax changed with upgrade to Cypress v9
- update add people tests as some terminology has changed

## To Test
- run `npm ci` (new package for Cypress tables added)
- add apiKey
- run `npm run cypress:runner -- --env configFile=dev`
- run tests for **add people**
- > tests should pass

Note cypress-get-tables has some vulnerabilities, added to ticket to investigate these

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
